contracts/ITitleEscrow.sol

```diff
@@ -11,6 +11,9 @@ interface ITitleEscrow {
   /// @dev This emits when the beneficiary endorsed the the holder's transfer.
   event TransferEndorsed(uint256 indexed _tokenid, address indexed _from, address indexed _to);
 
+  /// @dev This emits when the beneficiary endorsed the the holder's transfer.
```

Comment:
you copied the explanation above with the same typo :p

Reply:
haha thanks, i need to relook at the whole thing again, the nomenclature for the whole contract is weird

Reply:
fixed